From a4179e4c92b6365d39e09cb9cd63c476848013af Mon Sep 17 00:00:00 2001
From: "Christoph M. Becker" <cmbecker69@gmx.de>
Date: Wed, 27 Apr 2022 12:37:39 +0200
Subject: [PATCH] Fix test for curl 7.83.0
Patch-Source: https://github.com/php/php-src/commit/a4179e4c92b6365d39e09cb9cd63c476848013af

libcurl 7.83.0 removed some trailing exclamation marks from error
messages[1]; we have to cater to that.

[1] <https://github.com/curl/curl/commit/6968fb9d54dc3a1aaa1b16088f038eaf5dd8b2d7>
---
 ext/curl/tests/curl_basic_007.phpt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/curl/tests/curl_basic_007.phpt b/ext/curl/tests/curl_basic_007.phpt
index 3b53658d6a7e..3834e4674f82 100644
--- a/ext/curl/tests/curl_basic_007.phpt
+++ b/ext/curl/tests/curl_basic_007.phpt
@@ -20,5 +20,5 @@ curl_close($ch);
 
 ?>
 --EXPECTF--
-string(%d) "No URL set!%w"
+string(%d) "No URL set%A"
 int(3)
